.github/workflows/TagBot.yml

```
@@ -1,12 +1,17 @@
 name: TagBot
+
 on:
-  schedule:
-    - cron: '0 0 * * *'
+  issue_comment:
+    types:
+      - created
+  workflow_dispatch:
```

Match the latest [TagBot](https://github.com/JuliaRegistries/TagBot) setup example.
```suggestion
  workflow_dispatch:
    inputs:
      lookback:
        default: 3
permissions:
  contents: write
```